internal/pkg/cli/pipeline_init.go

```diff
@@ -359,24 +355,44 @@ func relPath(fullPath string) string {
 	return relPath
 }
 
-// TODO: Nice-to-have: have an opts.listRemoteRepos() method that execs out to `git remote -v` and parse repo name to offer select menu
-func (opts *InitPipelineOpts) selectGitHubRepo() error {
-	repo, err := opts.prompt.Get(
-		pipelineEnterGitHubRepoPrompt,
-		fmt.Sprintf(`The GitHub repository linked to your workspace. Pushing to this repository will trigger your pipeline build stage. Please enter full repository URL, e.g. "https://github.com/myCompany/myRepo", or the owner/rep, e.g. "myCompany/myRepo"`),
-		validateGitHubRepo,
+func (opts *InitPipelineOpts) getGitHubOwner() error {
+	owner, err := opts.prompt.Get(
+		pipelineEnterGitHubOwnerPrompt,
+		pipelineEnterGitHubOwnerHelpPrompt,
+		nil,
 	)
+	if err != nil {
+		return fmt.Errorf("fail to get GitHub owner name: %w", err)
```

Comment:
nit: Can we remove "fail to" from the error messages?
> When adding context to returned errors, keep the context succinct by avoiding phrases like "failed to", which state the obvious and pile up as the error percolates up through the stack

https://github.com/uber-go/guide/blob/master/style.md#error-wrapping